.github/issue_template.md

```diff
@@ -0,0 +1,35 @@
+## Problem Description
+
+---
+
+
+## Steps to Reproduce Problem
+
+
+---
+
+
+## Additional Info
+(Logs, screenshots, configuration, etc.)
+
+---
+
+## Pre-check
+- [ ] Confirmed in Vanille / mailing list / Slack?
+- [ ] Verified that it’s not already fixed or being worked on?
+
+---
+
+## Workflow
```

Comment:
1. Replicate the issue
2. Explore possible solutions and estimate time
3. Write a test or write it into manual testing scenarios
4. Code to Pass the Tests
5. Refactor & Review
6. Update documentation